packages/google-pay-integration/e2e/GooglePayBraintreeMockingResponses.ts

```diff
@@ -0,0 +1,23 @@
:null},"type":"PAYMENT_TYPE_API","initializationStrategy":{"type":"not_applicable"},"nonce":null,"initializationData":{"isAcceleratedCheckoutEnabled":false,"shouldRunAcceleratedCheckout":false,"isBraintreeVenmoEnabled":1,"intent":"capture","isComplete":false,"isCreditEnabled":true,"walletButtonStyle":{"color":"gold","shape":"rect","size":"medium","label":"checkout","height":40},"enabledLocalPaymentMethods":[],"availableLocalPaymentMethods":[]},"clientToken":"eyJ2ZXJzaW9uIjoyLCJhdXRob3JpemF0aW9uRmluZ2VycHJpbnQiOiJleUowZVhBaU9pSktWMVFpTENKaGJHY2lPaUpGVXpJMU5pSXNJbXRwWkNJNklqSXdNVGd3TkRJMk1UWXRjMkZ1WkdKdmVDSXNJbWx6Y3lJNkltaDBkSEJ6T2k4dllYQnBMbk5oYm1SaWIzZ3VZbkpoYVc1MGNtVmxaMkYwWlhkaGVTNWpiMjBpZlEuZXlKbGVIQWlPakUyT1RVek9ERXdPREFzSW1wMGFTSTZJbUl4Tmpsak1UZG1MV1JoWVRndE5HVTRZeTA0TVRkaUxXRTFZMk16TXpnNFpEUTJNU0lzSW5OMVlpSTZJbWhuWkdOa2JuRTBZell5ZVhsak5YSWlMQ0pwYzNNaU9pSm9kSFJ3Y3pvdkwyRndhUzV6WVc1a1ltOTRMbUp5WVdsdWRISmxaV2RoZEdWM1lYa3VZMjl0SWl3aWJXVnlZMmhoYm5RaU9uc2ljSFZpYkdsalgybGtJam9pYUdka1kyUnVjVFJqTmpKNWVXTTFjaUlzSW5abGNtbG1lVjlqWVhKa1gySjVYMlJsWm1GMWJIUWlPblJ5ZFdWOUxDSnlhV2RvZEhNaU9sc2liV0Z1WVdkbFgzWmhkV3gwSWwwc0luTmpiM0JsSWpwYklrSnlZV2x1ZEhKbFpUcFdZWFZzZENKZExDSnZjSFJwYjI1eklqcDdJbTFsY21Ob1lXNTBYMkZqWTI5MWJuUmZhV1FpT2lKaWFXZGpiMjF0WlhKalpTSjlmUS5OR0NBUURmQmE5OVNsSG1ZQWplQ2ExYzd1eTZiU2l3Qjdxd2Iyd0NWNDBDeUw0RXBqaGw5NE0xdVZFWU5yR1JZUG5vaTYtMF9QRFEzWjdlUF9DMkRrdyIsImNvbmZpZ1VybCI6Imh0dHBzOi8vYXBpLnNhbmRib3guYnJhaW50cmVlZ2F0ZXdheS5jb206NDQzL21lcmNoYW50cy9oZ2RjZG5xNGM2Mnl5YzVyL2NsaWVudF9hcGkvdjEvY29uZmlndXJhdGlvbiIsIm1lcmNoYW50QWNjb3VudElkIjoiYmlnY29tbWVyY2UiLCJncmFwaFFMIjp7InVybCI6Imh0dHBzOi8vcGF5bWVudHMuc2FuZGJveC5icmFpbnRyZWUtYXBpLmNvbS9ncmFwaHFsIiwiZGF0ZSI6IjIwMTgtMDUtMDgiLCJmZWF0dXJlcyI6WyJ0b2tlbml6ZV9jcmVkaXRfY2FyZHMiXX0sImNsaWVudEFwaVVybCI6Imh0dHBzOi8vYXBpLnNhbmRib3guYnJhaW50cmVlZ2F0ZXdheS5jb206NDQzL21lcmNoYW50cy9oZ2RjZG5xNGM2Mnl5YzVyL2NsaWVudF9hcGkiLCJlbnZpcm9ubWVudCI6InNhbmRib3giLCJtZXJjaGFudElkIjoiaGdkY2RucTRjNjJ5eWM1ciIsImFzc2V0c1VybCI6Imh0dHBzOi8vYXNzZXRzLmJyYWludHJlZWdhdGV3YXkuY29tIiwiYXV0aFVybCI6Imh0dHBzOi8vYXV0aC52ZW5tby5zYW5kYm94LmJyYWludHJlZWdhdGV3YXkuY29tIiwidmVubW8iOiJvZmYiLCJjaGFsbGVuZ2VzIjpbImN2diIsInBvc3RhbF9jb2RlIl0sInRocmVlRFNlY3VyZUVuYWJsZWQiOnRydWUsImFuYWx5dGljcyI6eyJ1cmwiOiJodHRwczovL29yaWdpbi1hbmFseXRpY3Mtc2FuZC5zYW5kYm94LmJyYWludHJlZS1hcGkuY29tL2hnZGNkbnE0YzYyeXljNXIifSwicGF5cGFsRW5hYmxlZCI6dHJ1ZSwicGF5cGFsIjp7ImJpbGxpbmdBZ3JlZW1lbnRzRW5hYmxlZCI6dHJ1ZSwiZW52aXJvbm1lbnROb05ldHdvcmsiOmZhbHNlLCJ1bnZldHRlZE1lcmNoYW50IjpmYWxzZSwiYWxsb3dIdHRwIjp0cnVlLCJkaXNwbGF5TmFtZSI6IkJpZ2NvbW1lcmNlIiwiY2xpZW50SWQiOiJBVnZYN0dwTzJELTJocUdaNW9fZWNJVHRxVTd3X1F1QTkxVHQyY29XWTNGdEhIX0JiWXdPZ3lwQmR5LVpFUmhKc1kwUTVMSkZFMm5ma3J4NiIsImJhc2VVcmwiOiJodHRwczovL2Fzc2V0cy5icmFpbnRyZWVnYXRld2F5LmNvbSIsImFzc2V0c1VybCI6Imh0dHBzOi8vY2hlY2tvdXQucGF5cGFsLmNvbSIsImRpcmVjdEJhc2VVcmwiOm51bGwsImVudmlyb25tZW50Ijoib2ZmbGluZSIsImJyYWludHJlZUNsaWVudElkIjoibWFzdGVyY2xpZW50MyIsIm1lcmNoYW50QWNjb3VudElkIjoiYmlnY29tbWVyY2UiLCJjdXJyZW5jeUlzb0NvZGUiOiJVU0QifX0=","returnUrl":null}';
```

Comment:
One thing we really don't like about mocking is exposing security details like token.
Could you please double check if we can replace tokens in this file with `*` or at least we should use a dummy one?

Reply:
@bc-peng Yes we can. Replaced to dummy one. Thanks!